File: cli/src/main/java/io/microconfig/cli/CliStarter.java
package io.microconfig.cli;

import static io.microconfig.cli.commands.CommandFactory.command;

public class CliStarter {
    public static void main(String[] args) {
        try {
            var code = command(args).execute();
            System.exit(code);
        } catch (CliException e) {
            System.out.println(e.getMessage());
            System.exit(e.getExitCode());
        }
    }
}

File: cli/src/main/kotlin/io/microconfig/cli/CliStarter.kt
package io.microconfig.cli

import io.microconfig.cli.commands.command
import kotlin.system.exitProcess

fun main(args: Array<String>) {
    try {
        val code = command(args).execute()
        exitProcess(code)
    } catch (e: CliException) {
        println(e.message)
        exitProcess(e.exitCode)
    }
}